src/edu/washington/escience/myria/CsvTupleWriter.java

```diff
@@ -58,6 +61,11 @@ private CsvTupleWriter(final OutputStream out, final CSVFormat csvFormat) throws
   }
 
   @Override
+  public void open(final OutputStream stream) throws IOException {
```

Comment:
I don't understand how this method plays with all the existing public constructors, which take an `OutputStream`. How is this class being instantiated if the `OutputStream` is only initialized in `open()`, and why do we still need the `OutputStream` argument to the constructors?